### PR TITLE
fix(ct): get rid of filters sidebar in OS

### LINKED ISSUE #5684: Category page :: Filtering options are no longer visible when filters are already active
## Environment
`localhost`
`dev`
`ct`
`next`

## Expected result
After filtering category page, customer can still see filter options and modify them

## Actual result
After filtering category page, filtering options disappear; only [Done] and [Clear All] buttons are displayed

## Steps to reproduce
1. Go to category page (_/c/<category>_)
1. Click [Filter]
1. Select a filter, e.g. color "white"
1. Click [Filter] again after data is reloaded/filtered - filter list is empty

## Recording
[^filters.webm]

### PR DESCRIPTION
**Pull request by lukaszjedrasik**
## Description

Based on [PR](https://github.com/vuestorefront/vue-storefront/pull/5822).
Includes two changes:
- (core) - added `CategoryPageHeader` and `FiltersSidebar` as a separate components,
- (ct) - removed `FiltersSidebar` in OS

## Related Issue

closes #5684 

## Motivation and Context

https://vsf.atlassian.net/browse/OS-47

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

#### Changelog
- [x] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [x] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [x] My code follows the code style of this project.

- [x] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        {{ $t('Filters') }}
+      </SfButton>
+    </LazyHydrate>
+
+    <div class="navbar__sort desktop-only">
+      <span class="navbar__label">{{ $t('Sort by') }}:</span>
+      <LazyHydrate on-interaction>
+        <SfSelect
+          :value="sortBy.selected"
+          placeholder="Select sorting"
```
Comment (filipsobol) [inline packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue:24]:
```suggestion
          :placeholder="$t('Select sorting')"
```

File: packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    <div class="navbar__view">
+      <span class="navbar__view-label desktop-only">{{ $t('View') }}</span>
+      <SfIcon
+        v-e2e="'tiles-icon'"
+        class="navbar__view-icon"
+        :color="isCategoryGridView ? 'black' : 'dark-secondary'"
+        icon="tiles"
+        size="12px"
+        role="button"
+        aria-label="Change to grid view"
```
Comment (filipsobol) [inline packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue:54]:
```suggestion
        :aria-label="$t('Change to grid view')"
```

File: packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        @click="changeToCategoryGridView"
+      />
+      <SfIcon
+        v-e2e="'list-icon'"
+        class="navbar__view-icon"
+        :color="!isCategoryGridView ? 'black' : 'dark-secondary'"
+        icon="list"
+        size="12px"
+        role="button"
+        aria-label="Change to list view"
```
Comment (filipsobol) [inline packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue:65]:
```suggestion
        :aria-label="$t('Change to list view')"
```

File: packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+          :placeholder="$t('Select sorting')"
+          class="navbar__select"
+          @input="th.changeSorting"
+        >
+          <SfSelectOption
+            v-for="option in sortBy.options"
+            :key="option.id"
+            :value="option.id"
+            class="sort-by__option"
+          >{{ option.value }}</SfSelectOption
```
Comment (dawid-ziobro) [inline packages/core/nuxt-theme-module/theme/components/CategoryPageHeader.vue:33]: Please fix formatting

File: packages/commercetools/theme/components/CategoryPageHeader.vue [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+    <div class="navbar__view">
+      <span class="navbar__view-label desktop-only">{{ $t('View') }}</span>
+      <SfIcon
+        v-e2e="'tiles-icon'"
+        class="navbar__view-icon"
+        :color="isCategoryGridView ? 'black' : 'dark-secondary'"
+        icon="tiles"
+        size="12px"
+        role="button"
+        :aria-label="$t('Change to grid view')"
```
Comment (dawid-ziobro) [inline packages/commercetools/theme/components/CategoryPageHeader.vue:18]: Please add this text and other new texts to translation files.

**Review by dawid-ziobro — APPROVED** [review]

**Pull request merged**